src/components/Layout/Navbar.vue

```diff
@@ -11,7 +11,20 @@
         <v-btn id="nav-user" text to="/user">Account</v-btn>
         <v-btn id="nav-logout" text to="/logout">Logout</v-btn>
       </template>
-      <template v-if="!isLoggedIn">
+      <v-menu bottom left v-if="$vuetify.breakpoint.xs && !isLoggedIn">
+        <template v-slot:activator="{ on, attrs }">
+          <v-btn icon v-bind="attrs" v-on="on">
+              <v-icon>menu</v-icon>
+          </v-btn>
+        </template>
+        <v-list>
+          <v-list-item>
+            <v-btn id="nav-login" text to="/login">Log in</v-btn>
```

Comment:
Question: Is there an easy way to deduplicate this and use the same elements as defined below?